### ISSUE TITLE
Compatibility with React-Native 0.56

### ISSUE THREAD

**Issue by Jeheonjeol** [GeekyAnts/react-native-easy-grid#68]
> error: bundling failed: Error: Unable to resolve module `react-native/Libraries/Renderer/shims/ReactNativePropRegistry` from `/ProjectName/node_modules/react-native-easy-grid/Utils/computeProps.js`: Module `react-native/Libraries/Renderer/shims/ReactNativePropRegistry` does not exist in the Haste module map

ReactNativePropRegistry no more exists after 0.56.0
https://github.com/facebook/react-native/commit/a8e3c7f5780516eb0297830632862484ad032c10

**Comment by akhil-geekyants** [comment]
@Jeheonjeol can you post your `package.json`

**Comment by Jeheonjeol** [comment]
@akhil-geekyants 
The following are modules and versions related this issue in my `package.json`
```
"react": "^16.3.1",
"react-native": "^0.56.0-rc.3",
"react-native-easy-grid": "^0.1.15"
```

Also, another GeekyAnts's library 'NativeBase' has same this issue.
https://github.com/GeekyAnts/NativeBase/issues/2035

**Comment by bobmulder** [comment]
Got the same issue which is caused by `react-native-easy-grid`. Any PR yet?

**Comment by SupriyaKalghatgi** [comment]
https://github.com/GeekyAnts/NativeBase/issues/2028

**Comment by SupriyaKalghatgi** [comment]
Fixed with 0.2.0
